R/features.R

```diff
@@ -4,7 +4,17 @@ features_config <- rlang::env(
 
 validate_extendr_features <- function(features, suppress_warnings) {
   features <- features %||% character(0)
-  vctrs::vec_assert(features, character())
+
+  is_char <- try(vctrs::vec_assert(features, character()), silent = TRUE)
```

Comment:
Why not `vctrs::vec_is(features, character())`?

Reply:
Because I didn't know about it! Will quickly push the change. Also, why not `inherits(features, "character")`?